Tests/TuistKitAcceptanceTests/ImplicitImportsLintCommandAcceptanceTests.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+import TuistAcceptanceTesting
+
+final class ImplicitImportsLintCommandAcceptanceTests: TuistAcceptanceTestCase {
+    func test_ios_app_with_frameworks() async throws {
```

Comment:
Let's add one extra acceptance test that does find some implicit dependencies. We already have a fixture for that: https://github.com/tuist/tuist/tree/main/fixtures/ios_app_with_implicit_dependencies